pkg/endpoints/pipeline_test.go

```diff
@@ -255,6 +255,28 @@ func TestTaskRun(t *testing.T) {
 		t.Errorf("Task2 is not returned: %s, %s", result.Items[0].Name, result.Items[1].Name)
 	}
 
+	// Test getAllTaskRuns function with name query
```

Comment:
Can we really not have any test for it because field selector doesn't work with fake? I preferred the previous approach if so where we just iterated